server/handlers_v2_test.go

```diff
@@ -96,7 +96,10 @@ func TestHTTPServer_ClustersDetailEndpointAggregatorResponseOk(t *testing.T) {
 			{
 				"cluster":"%v",
 				"cluster_name": "%v",
-				"last_checked_at":""
+				"last_checked_at":"",
+				"meta": {
+					"cluster_version": "4.7"
```

Comment:
Why don't you use [testdata.ClusterVersion](https://github.com/RedHatInsights/insights-results-aggregator-data/pull/204/files)? This way the value is not hardcoded :)

Reply:
Done